.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug
+about: Report a bug or confusing behavior
+title: ''
```

Comment:
"Bug: " like for feature request?

Reply:
Actually I removed the prefix from FR since labels will mark them accordingly